paymentsheet/src/test/java/com/stripe/android/paymentsheet/DefaultIntentConfirmationInterceptorTest.kt

```diff
@@ -88,6 +90,38 @@ class DefaultIntentConfirmationInterceptorTest {
         assertThat(confirmParams?.paymentMethodCreateParams).isEqualTo(createParams)
     }
 
+    @Test
+    fun `Returns confirm params with setup future usage set to off session when requires save on confirmation`() =
```

Comment:
It's a bit hard to tell from this test what the important variables here are. Can we simplify the test a little by extracting out the details that aren't actually relevant to this particular test case? (e.g. if the publishableKeyProvider is the same for multiple tests, just setting it in a global place instead of this individual test (though I'm not suggesting that you actually go through and update the other tests in this class)).